.travis.yml

```diff
@@ -1,3 +1,9 @@
 language: node_js
 node_js:
-  - "8.1.4"
\ No newline at end of file
+  - "8.1.4"
+before_install:
+  - wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+  - sudo dpkg -i google-chrome-stable_current_amd64.deb
+  - sudo apt-get install fonts-liberation libappindicator1 xdg-utils
+  - sudo apt-get install -f
+  - sudo apt-get install firefox xvfb
```

Comment:
These are the commands you would have to run in order to get the headless browser running.
- this means that these tests probably only pass on a linux machine.
- the cloud 9 space is a linux machine, and is setup to run these tests.
- this travis.yml is for continuous integration support (whenever we add it) through [travis](travis-ci.com)